assets/js/modules/analytics/components/setup/SetupFormGA4Transitional.js

```diff
@@ -51,12 +51,15 @@ export default function SetupFormGA4Transitional() {
 	const propertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
 	const profileID = useSelect( ( select ) => select( STORE_NAME ).getProfileID() );
 
+	const ga4PropertyID = useSelect( ( select ) => select( MODULES_ANALYTICS_4 ).getPropertyID() );
 	const ga4ExistingTag = useSelect( ( select ) => select( MODULES_ANALYTICS_4 ).getExistingTag() );
 	const ga4MeasurementID = useSelect( ( select ) => select( MODULES_ANALYTICS_4 ).getMeasurementID() );
 
 	const { setUseSnippet: uaSetUseSnippet } = useDispatch( STORE_NAME );
 	const { setUseSnippet: ga4SetUseSnippet } = useDispatch( MODULES_ANALYTICS_4 );
 
+	const shouldShowAssociatedPropertyNotice = propertyID || ga4PropertyID;
```

Comment:
See above, let's make this consistent with the other two scenarios: `accountID && accountID !== ACCOUNT_CREATE && ( propertyID || ga4PropertyID )`

Reply:
👍